smoke-test/test_system_info.py

```diff
@@ -0,0 +1,311 @@
+    main_response.raise_for_status()
+    main_data = main_response.json()
+    
+    assert "springComponents" in main_data, "Main endpoint should contain springComponents"
+    assert "properties" not in main_data, "Main endpoint should not contain properties (avoid duplication)"
+    
+    # Test properties endpoint - should contain detailed properties
+    props_response = auth_session.get(f"{auth_session.gms_url()}/openapi/v1/system-info/properties")
+    props_response.raise_for_status()
+    props_data = props_response.json()
+    
+    assert "properties" in props_data, "Properties endpoint should contain detailed properties"
+    assert "springComponents" not in props_data, "Properties endpoint should not contain springComponents"
+    
+    # Test spring components endpoint - should match main endpoint structure
+    components_response = auth_session.get(f"{auth_session.gms_url()}/openapi/v1/system-info/spring-components")
+    components_response.raise_for_status()
+    components_data = components_response.json()
+    
+    # Should have same structure as main endpoint's springComponents
+    assert main_data["springComponents"] == components_data, "Spring components endpoint should match main endpoint's springComponents"
+    
+    # Verify that GMS component has properties (from SpringComponentsCollector)
+    gms_info = main_data["springComponents"]["gms"]
+    assert "properties" in gms_info, "GMS component should have properties field with system properties"
+    assert isinstance(gms_info["properties"], dict), "GMS properties should be a dictionary"
+    
+    logger.info("Endpoint separation test passed - no duplication detected")
+
+
+def extract_api_token_from_session(session):
+    """Extract API token from frontend session cookies for OpenAPI authentication.
+    
+    Based on TestSessionWrapper pattern in tests/utils.py
+    """
+    # Extract actor URN from session cookies
+    actor_urn = session.cookies["actor"] 
+    
+    # Generate personal access token via GraphQL
+    json_payload = {
+        "query": """mutation createAccessToken($input: CreateAccessTokenInput!) {
+            createAccessToken(input: $input) {
+              accessToken
+              metadata {
+                id
+              }
+            }
+        }""",
+        "variables": {
+            "input": {
+                "type": "PERSONAL",
+                "actorUrn": actor_urn,
+                "duration": "ONE_HOUR",
+                "name": "Test API Token",
+                "description": "Token for smoke test authentication",
+            }
+        },
+    }
+    
+    response = session.post(f"{get_frontend_url()}/api/v2/graphql", json=json_payload)
+    response.raise_for_status()
+    
+    token_data = response.json()["data"]["createAccessToken"]
+    return token_data["accessToken"], token_data["metadata"]["id"]
+
+
+def test_system_info_authenticated_non_admin_user_returns_403():
```

Comment:
@RyanHolstien I added this test to make sure users without MANAGE_SYSTEM_OPERATIONS_PRIVILEGE cannot access the system-info endpoints. the test does create_user and remove_user in finally so normally there should not be users left after running this test. 

the pattern is similar to what privileges_and_test_user_setup from test_privileges.py (for example) is doing